tests/devapps/Telemetry/Telemetry.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
+    <SignAssembly>true</SignAssembly>
```

Comment:
You don't need to specify the details about signing here, because we have a top level Directories.Build.proj that does it.